src/value/index.rs

```diff
@@ -80,12 +80,7 @@ impl Index for str {
         }
         match *v {
             Value::Object(ref mut map) => {
-                // TODO: use entry() once LinkedHashMap supports entry()
-                // https://github.com/contain-rs/linked-hash-map/issues/5
-                if !map.contains_key(self) {
-                    map.insert(self.to_owned(), Value::Null);
-                }
-                map.get_mut(self).unwrap()
+                map.entry(self.to_owned()).or_insert(Value::Null)
```

Comment:
If I understand, this results in `self.to_owned()` being called when the entry already exists, while previously it was called only when the entry did not exist.